tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -220,8 +222,10 @@ def testFloatBasic(self):
   def testFloatTanhEdge(self):
     x = np.arange(40, 40 + 6).reshape(6).astype(np.float32)
     self._compareBoth(x, np.tanh, tf.tanh)
+    self._compareBothSparse(x, np.tanh, tf.tanh)
     x = np.arange(-40, -40 + 6).reshape(6).astype(np.float32)
     self._compareBoth(x, np.tanh, tf.tanh)
+    self._compareBothSparse(x, np.tanh, tf.tanh, thresh=-40)
```

Comment:
Yes, I think the point of this test is to check numerical stability of the Tanh for large values of x, so I chose to not sparsify.